internal/hcs/vmcompute.go

```diff
@@ -0,0 +1,549 @@
+package hcs
+
+import (
+	gcontext "context"
```

Comment:
Same here. Doesn't look like anything collides with `context`.

Reply:
It does. "context" is a parameter passed to some of the hcs* functions.